test/unit/gateways/shift4_v2_test.rb

```diff
@@ -0,0 +1,874 @@
+require 'test_helper'
+
+class Shift4V2Test < Test::Unit::TestCase
+  include CommStub
+
+  def setup
+    @gateway = Shift4V2Gateway.new(
+      secret_key: 'pr_test_SyMyCpIJosFIAESEsZUd3TgN'
```

Comment:
Same here, can this environment variable be visible on the web? Could you read from the .yml file?